README.md

```diff
@@ -146,14 +146,14 @@ playbook to install KubeVirt from given manifest.
 | ---------------- | ------------- | -------------------------------------- |
 | kubevirt\_mf     | string | Path to KubeVirt manifest, you can get one from [releases](https://github.com/kubevirt/kubevirt/releases) or build it from sources |
 | openshift\_ansible\_dir | string | Path to [OpenShift Ansible project][openshift-ansible-project] |
-| kconfig | string | Path to kubeconfig |
+| kubeconfig | string | Path to kubeconfig |
```

Comment:
@lukas-bednar Please elaborate where kubeconfig, a KubeVirt(?) configuration file, comes from. Then one won't question it and will not search for it.

Reply:
Addressed